components/locale-chooser.js

```diff
@@ -2,47 +2,137 @@ import React from 'react'
+  [mediaQueries.smallOnly]: {
+    position: 'relative',
+    width: '100%',
+  },
+}))
+
+const Item = glamorous.li((props, {colors, mediaQueries}) => ({
+  display: 'flex',
+  flex: 1,
+  textAlign: 'left',
+  margin: 0,
+  backgroundColor: colors.white,
+  lineHeight: 1,
+  transition: 'all .3s',
+  '&:focus, &:hover, &:active': {
+    backgroundColor: colors.primaryMed,
+  },
+  [mediaQueries.smallOnly]: {
+    textAlign: 'center',
+  },
+}))
+
+const Link = glamorous.a((props, {colors}) => ({
+  width: '100%',
+  padding: '6px 10px',
+  transition: 'all .3s',
+  '&:focus, &:hover, &:active': {
+    textDecoration: 'none',
+    color: colors.white,
+  },
+}))
+
+const localeContent = ({key, display}) =>
+  (<div>
+    {getFlag(key)} <span>{display}</span>
+  </div>)
+
+const localeItem = ({key, display}) =>
+  (<Item key={key}>
+    <Link href={localeToHref(key)}>
+      {localeContent({key, display})}
+    </Link>
+  </Item>)
 
-function LocaleChooser({locale, content, ...rest}) {
-  return (
-    <Select
-      value={locale}
-      onChange={changeLanguage}
-      {...rest}
-      aria-label="Locale selector"
-    >
-      <option value="en">en</option>
-      <option value="es">es</option>
-      <option value="fr">fr</option>
-      <option value="help">{content.help}</option>
-    </Select>
-  )
+class LocaleChooser extends React.Component {
+  state = {
+    open: false,
+  }
+
+  handleClick() {
+    this.setState(prevState => {
+      return {open: !prevState.open}
+    })
+  }
+
+  render() {
+    return (
+      <Wrapper>
+        <Toggle onClick={this.handleClick.bind(this)} isOpen={this.state.open}>
+          {localeContent(locales.find(({key}) => key === this.props.locale))}
+        </Toggle>
+        {this.state.open &&
+          <List aria-label="Locale selector">
+            {locales.map(localeItem)}
+            {localeItem({key: 'help', display: this.props.content.help})}
+          </List>}
+      </Wrapper>
+    )
+  }
 }
+export default withContent({component: 'locale-chooser'}, LocaleChooser)
 
-function changeLanguage({target: {value}}) {
-  let url
-  if (supportedLocales.includes(value)) {
+function localeToHref(locale) {
+  if (supportedLocales.includes(locale)) {
     const {host} = getLocaleAndHost()
     const {protocol, pathname, hash, search} = window.location
-    const prefix = fallbackLocale === value ? '' : `${value}.`
-    url = `${protocol}//${prefix}${host}${pathname}${search}${hash}`
-  } else {
-    url =
-      'https://github.com/kentcdodds/glamorous-website/blob/master/other/CONTRIBUTING_DOCUMENTATION.md'
+    const prefix = fallbackLocale === locale ? '' : `${locale}.`
+    return `${protocol}//${prefix}${host}${pathname}${search}${hash}`
+  }
+
+  return 'https://github.com/kentcdodds/glamorous-website/blob/master/other/CONTRIBUTING_DOCUMENTATION.md'
+}
+
+function getFlag(locale) {
+  const style = {width: '1em', height: '100%'}
+  switch (locale) {
```

Comment:
There must be a better option... I tried with `require('./svgs/${locale}.svg')` but kept bumping into problems.